scripts/sexbound/lib/sexbound/sextalk.lua

```diff
@@ -95,6 +95,10 @@ end
 
 --- Function to choose a random available 
 function Sexbound.Sextalk:triggerTalk()
+
+    -- disable dialog if its a defeat node (will have separate dialog in the future)
+    if self._parent._containsDefeated then return end
```

Comment:
Certainly a workaround hack, so please ensure to properly mark down that this needs changing in the future.

Reply:
I think you mentioned a process that was something along the lines of duplicating the existing positions and making "defeat" versions with their own dialog. I haven't actually looked into the code to find where I'd have to make changes to do that yet but it seemed too complicated for my fried brain so I just added this for now.

Reply:
On the main node objects themselves, the available positions are defined.
```json
"sexboundConfig" : {
    "animationPartsCentered": false,
    "position" : {
      "sex" : [ "standing", "doggy", "missionary", "cowgirl", "facesitting", "butterfly", "fellatio", "reverse_cowgirl" ]
    }
  }
```

You'd change these to different one. I'd just rename them all to "*_defeat". These new positions also need to be defined in `/positions/positions.config`. That file maps the name to the actual file path. Then just add the new positions. 

In the new position configs, you would mostly just define `"base" : "<base position name>"` to have ALL of the base position's data, then just overwrite the `"dialog": {}` table to link to new defeat specific dialog files for each race.

Reply:
That's pretty straightforward. I'll add in that functionality. Writing the actual dialog however isn't as much in my wheelhouse so I'd leave those free to be filled in if that's alright.